__tests__/components/DurationInput.spec.ts

```diff
@@ -100,9 +102,43 @@ describe('components/DurationInput', () => {
         const chosenValue = '20000';
         const wrapper = getDurationInputWrapper({ showRelativeTime: true, targetAsset: 'namespace', value: '10000' });
         const component = wrapper.vm as DurationInputTs;
+
+        // Act:
         component.chosenValue = chosenValue;
 
         // Assert:
         expect(wrapper.emitted('input')[0][0]).toBe(chosenValue);
     });
+
+    // TODO test for validationRule
+    test('duration validation rule when the target asset is mosaic', async () => {
+        // Arrange:
+        const wrapper = getDurationInputWrapper({ targetAsset: 'mosaic', value: '10000' });
+        const component = wrapper.vm as DurationInputTs;
+
+        // Act:
+        const durationValidationRule = component.validationRule;
+
+        // Assert:
+        expect(durationValidationRule).toContain(
+            `max_value:${networkConfig[NetworkType.TEST_NET].networkConfigurationDefaults.maxMosaicDuration}`,
+        );
+    });
+
+    test('duration validation rule when the target asset is namespace', async () => {
+        // Arrange:
+        const wrapper = getDurationInputWrapper({ targetAsset: 'namespace', value: '10000' });
+        const component = wrapper.vm as DurationInputTs;
+
+        // Act:
+        const durationValidationRule = component.validationRule;
+
+        // Assert:
+        expect(durationValidationRule).toContain(
+            `min_value:${
+                networkConfig[NetworkType.TEST_NET].networkConfigurationDefaults.minNamespaceDuration /
```

Comment:
why dividing by blockGenerationTargetTime ? is input a block count or a human readable duration ?

Reply:
here is how it's been calculated: https://github.com/symbol/desktop-wallet/blob/48a69cf1c0719cee71f04dbbcf8c5736ee6c9a99/src/core/validation/ValidationRuleset.ts#L47

And this is how it's presented in the UI (both block count and human readable value)
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/6256269/183941458-ce614498-cac6-45f9-9f63-90685ad94af8.png">

I think it's the block count (86400 * 30 secs = 30 days)

Reply:
do you know where/how the wallet is pulling the configuration from the network?

Reply:
asking because in the config file for the server, the setting looks like this:
```
minNamespaceDuration = 30d
```

Reply:
Sorry, somehow I missed this comment.

I think it's here, getting from the network/properties endpoint:
https://github.com/symbol/desktop-wallet/blob/48a69cf1c0719cee71f04dbbcf8c5736ee6c9a99/src/services/NetworkService.ts#L77

And merging with the defaults:
https://github.com/symbol/desktop-wallet/blob/48a69cf1c0719cee71f04dbbcf8c5736ee6c9a99/src/services/NetworkService.ts#L133

And yes, you're right, it's returning 30d from the endpoint 🤔 
I will look into it deeper tomorrow.

Reply:
When I checked it, I found out that `30d` is being converted to seconds in:
https://github.com/symbol/desktop-wallet/blob/af3c661ed6c921b2528dc55cfb1c16ad13ff6dab/src/core/utils/NetworkConfigurationHelpers.ts#L160

So both the calculation and the presentation seem to be fine (also had @cryptoBeliever looked at it). There was one issue that the validation was always using `testnet` configurations, I fixed that issue and updated the tests accordingly.